CHANGELOG.md

```diff
@@ -6,511 +6,514 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/)
 
 The types of changes are:
 
-* `Added` for new features.
```

Comment:
Is this something your IDE has done automatically?

Reply:
Apparently so, I didn't even notice it did this. I'll see if I can find what happened.